### ISSUE TITLE
CommandLine.Parser.Default.FormatCommandLine incorrectly shows DateTime/DateTime? option

### ISSUE THREAD

**Issue by nknoop** [commandlineparser/commandline#502]
When you call the FormatCommandLine DateTime and DateTime? are not formatted correctly (if you feed the generated string back to the CommandLine it shows an error)

```
internal class Options
{
	[Option(
		longName: "start",
		Default = null,
		HelpText = "Some help text"
	)]
	public DateTime? Start { get; set; }
}

```
Formatted date looks something like this for me: 14/08/2019 09:51:12

There seem to be missing " quotations around it (or a T between the date and the time) also the formatting of the date is wrong, should be 2019-08-14

**Comment by moh-hassan** [comment]
@nknoop 
Can you try the nuget dev Package on [Ci Server](https://ci.appveyor.com/api/buildjobs/iy7242cxnac19owf/artifacts/src%2FCommandLine%2Fbin%2FRelease%2FCommandLineParser.2.7.0-alpha-48-hgkdqqrq.nupkg) to know if your issue is resolved.

**Comment by moh-hassan** [comment]
[online demo](https://dotnetfiddle.net/8gPgBK) to show the fix in v2.7+
